gateway/src/project.rs

```diff
@@ -201,10 +201,6 @@ impl Project {
         }
     }
 
-    pub fn create(project_name: ProjectName) -> Self {
-        Self::Creating(ProjectCreating::new_with_random_initial_key(project_name))
-    }
-
```

Comment:
(3rd commit proposal, can be removed if not liked)
Removing this public method as there is already a builder pattern for the inner `ProjectCreating`.
Alternatively one could change this method to accept a `fqdn: Option<String>` as well, to reflect the current usage of the method - I prefer the former though.